### ISSUE TITLE
Sidebar on albums does not show other tabs

### ISSUE THREAD

**Issue by pulsejet** [pulsejet/memories#464]
The sidebar is a reduced metadata sidebar on external shares and albums. This prevents collaboration.

**Comment by afonsoc12** [comment]
Hi,
Is there an ETA for this issue?
Thanks

**Comment by pulsejet** [comment]
No, it's blocked by upstream (nextcloud/server). Can't fix it here.

**Comment by thundervortex** [comment]
Hi,
While testing on Nextcloud 33 / Memories 8.0.1, I believe I may have additional diagnostic information that could help narrow this down…. I’m not sure if this is the same underlying issue or something new, but I wanted to share some findings.

**Environment**
- Nextcloud Hub 10 / Nextcloud 33.0.5
- Memories 8.0.1
- Docker AIO

**Setup**
- I am not viewing an album or an external share.
- During testing I reproduced the issue using the default Nextcloud sample image (Library.jpg) stored in my personal Photos folder to eliminate Team Folders as a variable.
- Comments, Activity, Chat, Sharing, and Versions all work correctly in the Files app.

**Issue**
Opening the exact same file in Memories consistently shows only the reduced metadata sidebar.

**Debugging**
While investigating in the browser console I found:

globalThis.OCA.Files.Sidebar
returns => undefined

However,

globalThis.OCA.Files._sidebar()
returns => a populated sidebar object.

While browsing the source, I noticed what may be a related reference in Sidebar.vue, although I don’t know whether it’s relevant.

globalThis.OCA.Files.Sidebar

I’m not familiar enough with the codebase to know whether this is intentional, but I wanted to mention it in case it’s relevant.
<img width="1439" height="1211" alt="Image" src="https://github.com/user-attachments/assets/a54faea2-b5ea-47d8-b9fc-b550406972d6" />

**Comparison**
The attached screenshots show the exact same file:
- Files app — full collaboration sidebar (Sharing, Chat, Activity, Versions)
<img width="1083" height="904" alt="Image" src="https://github.com/user-attachments/assets/3934eaf1-860e-4f70-9950-c488b9e59185" />

- Memories — reduced metadata sidebar only
<img width="1280" height="956" alt="Image" src="https://github.com/user-attachments/assets/007e1635-08b1-4565-9b89-a7078167c9e7" />

Since Issue #464 mentions this was previously blocked upstream by Nextcloud, I wasn’t sure if this is expected behavior or whether Nextcloud 33’s sidebar API has changed.

**Additional observation:**
The sidebar itself is created (.app-sidebar exists), however no collaboration tabs are ever registered.
Running:

document.querySelectorAll(".app-sidebar-tab")
Returns => NodeList(0)

whereas the Files app for the exact same file displays the expected Sharing / Chat / Activity / Versions tabs.

The issue appears reproducible even with a completely standard Nextcloud file, so it doesn’t appear to be related to Team Folders, permissions, or sharing configuration.  Happy to test anything that would help narrow this down.
